react-components/src/data-providers/core-dm-provider/check3dConnectedEquipmentQuery.ts

```diff
@@ -144,6 +67,54 @@ function getRevisionsPointCloudVolumes(object3dTableName: string): QueryTableExp
   };
 }
 
+type Image360ConnectionExpressions<Prefix extends string> = Record<
+  `${Prefix}_360_annotation_edges` | `${Prefix}_360_image_nodes`,
+  QueryTableExpressionV3
+>;
+
+function getRevisionsImage360Annotations<const Prefix extends string>(
+  prefix: Prefix,
+  object3dTableName: string,
+  revisionRefs: DmsUniqueIdentifier[]
+): Image360ConnectionExpressions<Prefix> {
+  const edgesResultExpression = `${prefix}_360_annotation_edges`;
+  const imagesResultExpession = `${prefix}_360_image_nodes`;
+
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+  return {
+    [edgesResultExpression]: {
+      edges: {
+        from: object3dTableName,
+        direction: 'outwards',
+        filter: { hasData: [COGNITE_IMAGE_360_ANNOTATION_SOURCE] }
+      }
+    },
+    [imagesResultExpession]: {
+      nodes: {
+        from: edgesResultExpression,
+        chainTo: 'destination',
+        filter: {
+          and: [
+            {
+              hasData: [COGNITE_IMAGE_360_SOURCE]
+            },
+            {
+              in: {
+                property: [
+                  COGNITE_IMAGE_360_SOURCE.space,
+                  `${COGNITE_IMAGE_360_SOURCE.externalId}/${COGNITE_IMAGE_360_SOURCE.version}`,
+                  'collection360'
+                ],
+                values: revisionRefs
+              }
+            }
+          ]
+        }
+      }
+    }
+  } as Image360ConnectionExpressions<Prefix>;
```

Comment:
Incorrect type? Can we get rid of this `as`?

Reply:
See comment above

Reply:
(This was also removed)